app/scripts/views/mixins/signed-out-notification-mixin.js

```diff
@@ -34,11 +33,9 @@ define(function (require, exports, module) {
     },
 
     navigateToSignIn () {
-      this.navigate('signin', {
-        success: BaseView.t('Signed out successfully')
-      }, {
-        clearQueryParams: true
-      });
+      // Navigate via the back-end in order to regenerate
+      // flow id and flow begin time for the next session.
+      this.window.location.href = '/signin';
```

Comment:
> > Do you mean for the general case, and then override that in my usage, or do you mean my usage itself should propagate query params?
>
> Only in this case.

A side-effect of this would be propagating the resume token. The resume token takes precedence over the data attributes for flow model initialisation. So if we propagate the resume token, it will cause duplicate flow events to be emitted, which is the problem I want to fix here.

I could make that resume-token-or-data-attribute check dependent on the current view, but that feels a bit sketchy. Any other suggestions?